extensions/amp-playbuzz/0.1/amp-playbuzz.css

```diff
@@ -0,0 +1,104 @@
+/**
+ * Copyright 2015 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+.pb_feed_loading {
+        height: 320px;
+        overflow: hidden;
+}
+
+.pb_top_content_container {
+        opacity: 0;
+}
+
+.pb_top_content_container {
+        opacity: 1;
+}
+
+.pb_feed_placeholder_container {
+        position: absolute;
```

Comment:
that's taking care of the internal arrangement of the placeholder + loading indicator